MAINTAINERS.md

```diff
@@ -0,0 +1,26 @@
+# Maintainers
+
+This document is for project maintainers who are doing versioning, releases, and publishing for the cardstack mono-repo.
+
+## Updating package versions
+
+The versions of all packages in the mono-repo are updated at the same time, to the same version number.
+
+1. Get the latest code on master and create a branch. `git checkout master`, `git pull origin master`, `git checkout -b vX.Y.Z`
+2. Look at the `git log` since the last release, and list anything significant in the `CHANGELOG.md`.
+Determine whether the release should be a major (breaking), minor, or patch.
+For breaking releases, include upgrade instructions in the changelog.
+3. Commit the changes, make sure your working tree is clean, and push your branch to GitHub.
+4. To update versions in the `package.json` for all packages, use `npx lerna version --force-publish="*" --exact`. `npx` will use the version of `lerna` that is installed as a `devDependency` of the mono-repo. `--force-publish` will update interdependencies as well as the version numbers of the packages themselves.
```

Comment:
Is lerna smart enough to know which version number to set? Or does that correlate with the branch name?

Reply:
It checks the current version, then prompts you to choose major/minor/patch, and displays the options that increment. I'll add a line about this. Thanks for the feedback!